CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ x.y.z Release notes (yyyy-MM-dd)
 * Add ability to stream change events on a remote MongoDB collection with `[RLMMongoCollection watch:delegate:delegateQueue]`, `MongoCollection.watch(delegate)`. When calling `watch(delegate)` you will be given a `RLMChangeStream` (`ChangeStream`), this will be used to invalidate and stop the streaming session by calling `[RLMChangeStream close]` (`ChangeStream.close()`) when needed.
 * Add `MongoCollection.watch`, which is a Combine publisher that will stream
   change events each time the remote MongoDB collection is updated.
+* Add Combine support for: PushClient, APIKeyAuth, User.Functions with dynamicMemberLookup, MongoCollection, EmailPasswordAuth, App.login
```

Comment:
Also, on top of what @bmunkholm said, this is also missing some of the added functionality (notably on the `User`)